common/archiver/README.md

```diff
@@ -45,13 +46,22 @@ type HistoryArchiver interface {
 
 ```go
 type VisibilityArchiver interface {
+    // Archive is used to archive one workflow visibility record. Check the Archive() method of the HistoryArchiver interface for parameters' meaning and requirements for this method. 
+    // The only difference is that the ArchiveOption parameter won't include an option for recording process. 
+    // Please make sure your implementation is lossless. If any in-memory batching mechanism is used, then those batched records will be lost during server restarts. 
+    // This method will be invoked when workflow closes. This means if conflict resolution happens after a workflow closes, this method will be called more than once for the same workflow execution.
```

Comment:
// This method will be invoked when workflow closes. Note that because of conflict resolution, it is possible for a workflow to through the closing process multiple times, which means that this method can be invoked more than once after a workflow closes.